single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java

```diff
@@ -633,14 +630,13 @@ protected void incorporateComponents(String prefixId, Node node, Element g, Diag
                 && ((ExternCell) node.getCell()).getDirection() == BusCell.Direction.BOTTOM;
 
         if (node instanceof Fictitious3WTNode) {
```

Comment:
Also here, we this refactoring we should be able to remove all 3wt specificity in SVG writer. That would allow some user to add another multi-part component without changing the code of SVG writer.

Reply:
The insertion of SVG component node has been simplified in DefaultSVGWriter (no more test of the 2wt, 3wt or inductor node type like before)